THIRD-PARTY

```diff
@@ -1,6 +1,7 @@
-** aws-cdk -- https://github.com/aws/aws-cdk
-** jsii -- https://github.com/aws/jsii
-** MongoDB NodeJs Driver -- https://github.com/mongodb/node-mongodb-native
+** aws-cdk; version 1.52.0 -- https://github.com/aws/aws-cdk
```

Comment:
Would be nice to remove the version numbers from these. They change almost every RFDK release, and we don't keep them up-to-date.